addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -462,6 +494,7 @@ export class PropertiesField extends Component {
             {
                 readonly: this.props.readonly || !this.state.canChangeDefinition,
                 canChangeDefinition: this.state.canChangeDefinition,
+                canModifyTags: () => this.canModifyPropertiesDefinition(),
```

Comment:
can you keep the name `canModifyPropertiesDefinition` ? (so no `canModifyTags`, easier to grep and follow the path)

also I would call it `checkDefinitionWriteAccess` or something like that, to not conflict to much with `canChangeDefinition`